src/drivers/abstract/spellbook.js

```diff
@@ -574,7 +571,7 @@ function formatDelete(spell) {
 function formatConditions(spell, conditions) {
   return conditions
     .map(condition => {
-      return isLogicalOp(condition) && condition.name == 'or' && conditions.length > 1 && condition.args.length > 1
+      return isLogicalOp(condition) && condition.name == 'or' && conditions.length > 1
```

Comment:
spellbook.js reverted

Reply:
oh, btw, if we intend to start abstracting SQL manipulation code, I think starting with spellbook is a good idea.